index.html

```diff
@@ -1,433 +1,260 @@
+            <span>Languages</span>
+            <img src="imgs/Icon-Down.png" alt="Open and Unhide">
+          </div>
 
+          <div class="row">
+            <div class="lang1 col-2">
+              <img src="imgs/JS.png" alt="Javascript">
+              <span class="lang2">javaScript</span>
+            </div>
+            <div class="lang1 col-2">
+              <img src="imgs/HTML.png" alt="HTML">
+              <span class="lang2">HTML</span>
+            </div>
+            <div class="lang1 col-2">
+              <img src="imgs/CSS.png" alt="Css">
+              <span class="lang2">Css</span>
+            </div>
+          </div>
 
-        <ul class="lang-section hg">
-          <li>
-            Frameworks
-            <img src="imgs/Icon-Right.png" alt="" />
-          </li>
-          <li>
-            Skills <img src="imgs/Icon-Right.png" alt="" />
-          </li>
-        </ul>
-      </section>
+          <div class="flexbox-item4 flexbox-item4-1">
+            <span>Frameworks</span>
+            <img src="imgs/Icon-Right.png" alt="Right Icon">
+          </div>
+          <img src="imgs/Separator_Bottom.png" alt="Bottom line">
+          <div class="flexbox-item4 flexbox-item4-1">
+            <span>Skills</span>
+            <img src="imgs/Icon-Right.png" alt="Icons">
+          </div>
+          <img src="imgs/Separator_Bottom.png" alt="Bottom line">
 
+        </div>
+      </div>
     </div>
 
-    <footer>
-      <h3>Contact me</h3>
-      <p>
-        If you have an application you are interested in developing, 
-        a feature that you need built or a project that needs coding. 
-        I’d love to help with it 
-      </p>
-      <form action="https://formspree.io/f/mqknglqn" method="post">
+  </main>
+  <footer>
+    <div class="contact-container">
+      <div class="contact-item-1">
+        <span>Contact me</span>
+      </div>
+      <div class="contact-item-2">
+        <p>If you have an application you are interested in developing, a feature that you need
+          built or a project that needs coding. I’d love to help with it </p>
+      </div> <br>
+      <div class="contact-item-3">
+        <form action="https://formspree.io/f/mqknglqn" method="post">
           <input class="form" type="text" name="username" maxlength="30" placeholder="Your name" required> <br>
           <input class="form" type="email" name="useremail" placeholder="Enter Email" required> <br>
-          <textarea class="form" cols="30" name="description" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>       
-          <button type="submit">Get it Touch</button> <br>
-      </form>
-      <div class="end-line"><img src="imgs/end-line.png" alt="end-line"></div> <br>
-    </footer>
-  </div>
+          <textarea class="form" cols="30" name="description" rows="10" placeholder="Write your message here"
+            maxlength="500" required></textarea> <br>
```

Comment:
- [ ] Hey, Please kindly add the label tag to your form input and textarea.  A form should have the label tag as the label tag point or specify a label for an <input>, <select>, <textarea> element of a form to improve accessibility for screen users. There are many ways to add the label tag. use this for yours

`<label for="username"><input type="text" name="username" id="username" maxlength="30" placeholder="username" required></label>`